src/index.html

```diff
@@ -11,7 +11,193 @@
+          Full date of birth:
+          <input
+            type="date"
+            name="dateOfBirth"
+            min="1923-01-01"
+            max="2022-01-01"
+          >
+        </label>
+        <label>
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <label class="form-label">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+        <label class="form-label2">
+          Password:
+          <input
+            type="password"
+            name="password"
+            id="password"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <label class="form-label">
+          Do you love cats?
+          <label class="radio-label">
+            <input
+              type="radio"
+              name="cat"
+              value="Yes"
+              required
+            >
+            Yes
+          </label>
+          <label class="radio-label">
+            <input
+              type="radio"
+              name="cat"
+              value="No"
+              required
+            >
+            No
+          </label>
+        </label>
+        <label class="form-label">
+         What is your favorite color?
+          <input
+            type="color"
+            name="color"
+            required
+          >
+        </label>
+        <label class="form-label">
+          What time do you go to bed?
+          <input
+            type="time"
+            name="time"
+            required
+          >
+        </label>
+        <label class="form-label">
+          What are your favorite brand of cars?
+          <select
+            class="choise"
+            name="favorite brand"
+            multiple
+          >
+            <option
+              value="BMW"
+            >
+              BMW
+            </option>
+            <option
+              value="Audi"
+            >
+              Audi
+            </option>
+            <option
+              value="Lada"
+            >
+              Lada
+            </option>
+          </select>
+        </label>
+        <label
+          class="facts"
+        >
+          How do you rate our work?
+          <input
+            name="work rate"
+            type="range"
+            value="0"
+          >
+        </label>
+      </fieldset>
+      <fieldset>
+        <legend>Additional info:</legend>
+        <label class="form-label">
+          Comments:
+          <textarea
+            class="info"
+            name="Comments"
+            cols="20"
+            rows="2"
+          >
+          </textarea>
```

Comment:
```suggestion
          ></textarea>
```
to prevent this
<img width="303" alt="image" src="https://user-images.githubusercontent.com/53611910/233841824-ceef2ec8-74ba-44ec-88e8-bc62add15ebf.png">